kalite/distributed/static/js/distributed/utils/api.js

```diff
@@ -76,7 +76,9 @@ function handleFailedAPI(resp, error_prefix) {
             try {
                 messages = $.parseJSON(resp.responseText || "{}").messages || $.parseJSON(resp.responseText || "{}");
             } catch (e) {
-                var error_msg = sprintf("%s<br/>%s<br/>%s", resp.status, resp.responseText, resp);
+                // Replacing resp.responseText with "blah blah blah" is just a workaround... this should be fixed.
+                // See https://github.com/learningequality/ka-lite/issues/4203
+                var error_msg = sprintf("%s<br/>%s<br/>%s", resp.status, "blah blah blah", resp);
```

Comment:
Removing the "blah blah blah" but on second thought I stand by the discussion in #4203... printing the `responseText` is potentially very ugly.